### PR TITLE
BZ1892213: Added an important note

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.5+
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1892213
QE Approval: @jinyunma 
Preview Link: https://deploy-preview-31311--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-requirements-user-infra_installing-vsphere and https://deploy-preview-31311--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm.html#installation-requirements-user-infra_installing-ibm-z-kvm

### CODE REVIEW

File: modules/installation-requirements-user-infra-ibm-z-kvm.adoc [hunk 1]
```diff
@@ -35,6 +35,11 @@ you install the cluster.
 To improve high availability of your cluster, distribute the control plane machines over different {op-system-base} instances on at least two physical machines.
 ====
 
+[IMPORTANT]
+====
+All virtual machines must reside in the same data store and in the same folder as the installer.
```
Comment (bobfuru) [inline modules/installation-requirements-user-infra-ibm-z-kvm.adoc:40]: s/data store/datastore
^This is a VMware/vSphere thing (even though I know ISG suggests two words).

Comment (bobfuru) [reply modules/installation-requirements-user-infra-ibm-z-kvm.adoc:40]: Avoid having stacked admonitions like this if possible. (It's not even ideal that there are already 2 stacked, but adding a third should be avoided.)

What about moving this addition to the end of the module?

File: modules/installation-requirements-user-infra-ibm-z-kvm.adoc [hunk 2]
```diff
@@ -35,6 +35,11 @@ you install the cluster.
...
+====
+
 The bootstrap, control plane, and compute machines must use the {op-system-first} as the
```
Comment (bobfuru) [inline modules/installation-requirements-user-infra-ibm-z-kvm.adoc:43]: While you're in this file, would you mind making this change, please?
s/must use the/must use/

**Review by bobfuru — COMMENTED** [review]
Suggestion to split up the stacking of admonition notes, and a couple of other minor requests. Thanks!

**Comment by bobfuru** [comment]
LGTM!

**Comment by jinyunma** [comment]
LGTM for doc related with user-provisioned infrastructure on vSphere/VMC.
I'm not familiar with "Installing a cluster with RHEL KVM on IBM Z and LinuxONE", so could not verify this doc.

**Comment by bobfuru** [comment]
@holgwolf Could you PTAL for QE review of changes to `modules/installation-requirements-user-infra-ibm-z-kvm.adoc` file?

**Comment by SNiemann15** [comment]
@bobfuru @sagidlow   I also have a [PR31155](https://github.com/openshift/openshift-docs/pull/31155) open that pertains to the same two files and the non KVM file contains an error that corrupts the min resource reqs table if someone makes a change to the file. There was a lengthy discussion on this in [#forum-docs](https://coreos.slack.com/archives/C2ZA5QGMV/p1617268603069800) last Thursday and I resolved the error in my PR. 
Should I get my PR merged first so that the table doesn't get corrupted? 

Also I'm not even sure if the note you added applies to the z/VM install on IBM Z. I've reached out to @holgwolf and @wvoesch to verify that for both KVM and z/VM.

**Comment by holgwolf** [comment]
@bobfuru this patch does not apply for Z and should not be added to any z specific documentation.

**Comment by vikram-redhat** [comment]
I just reviewed the BZ and if I am not mistaken, it only pertained to vSphere docs. Is the change in the https://github.com/openshift/openshift-docs/pull/31311/files#diff-07f06909facd1cc2de921b6c0cadccfe172217e739de0ee642c03a01fa4425b4 file added by mistake? We may also need to conditionalize this text so it only appears in the vsphere docs as installation-requirements-user-infra.adoc file is a module used by almost all install docs.

**Comment by SNiemann15** [comment]
@manojnkumar Please take a look if this note applies to IBM Power or not. 
cc: @ktania46

**Comment by bobfuru** [comment]
Hi @sagidlow - I do agree with @vikram-redhat that this should not be applied to P/Z docs, so any changes to `modules/installation-requirements-user-infra-ibm-z-kvm.adoc` should be reverted/not applied. 

And I also agree that a conditional `ifdef::vsphere[]` with `ifeval` should be used in the `installation-requirements-user-infra.adoc` file to keep this statement only within vSphere docs UPI docs. Let me know if you would like to discuss. Thanks!

**Comment by sagidlow** [comment]
@bobfuru  - I have reverted the change to add the note. I did leave the grammar change in the file though. Let me know if you want me to revert that as well.

File: modules/installation-requirements-user-infra.adoc [hunk 3]
```diff
@@ -82,6 +91,13 @@ Note that {op-system} is based on {op-system-base-full} 8 and inherits all of it
 endif::[]
 See link:https://access.redhat.com/articles/rhel-limits[Red Hat Enterprise Linux technology capabilities and limits].
 
+[IMPORTANT]
+====
+ifdef::vsphere[]
+All virtual machines must reside in the same datastore and in the same folder as the installer.
+endif::vsphere[]
+====
+
```
Comment (bobfuru) [inline modules/installation-requirements-user-infra.adoc:100]: Move `ifdef` and `endif` to wrap the entire admonition block:

```
ifdef::vsphere[]
[IMPORTANT]
====
All virtual machines must reside in the same datastore and in the same folder as the installer.
====
endif::vsphere[]
```

**Comment by bobfuru** [comment]
Updates LGTM

**Pull request merged**